pyproject.toml

```diff
@@ -62,6 +62,10 @@ wapiti = "wapitiCore.main.wapiti:wapiti_asyncio_wrapper"
 wapiti-getcookie = "wapitiCore.main.getcookie:getcookie_asyncio_wrapper"
 
 [project.optional-dependencies]
+ssl = [
+        "humanize==4.4.0",
+        "sslyze==5.1.1"
```

Comment:
I am not sure but maybe you also need `pydantic` ?

Reply:
@devl00p so this line `"pydantic==1.10.2" # This is to avoid a conflict with sslyze` is not needed anymore in `test` dependencies just below ?

Reply:
It seems it works with recent versions of pydantic (1.10.7) so it should be ok